app/views/product/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="container-fluid text-center">
+
+  <h1>List of Products</h1>
+  <br><br>
+
+  <table class="table table-hover">
+    <thead>
+      <tr>
+        <th>ID</th>
+        <th>Title</th>
+        <th>Action</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @products.each do |product| %>
+        <tr>
+          <td><%= product.id %></td>
+          <td><%= product.title %></td>
+          <td>
+            <a href="/product/<%= product.id %>" class="btn btn-sm btn-outline-success">View</a>
```

Comment:
@Krutik31 You are creating a link here, so use rails method link_to
Check it here how can a link be created in rails
https://www.rubyguides.com/2019/05/rails-link_to-method/